### PR TITLE
Add provision to store partition table in FLASH

### PR DESCRIPTION
**Pull request by btheosam**
Bootloader needs to know about the partitions to perform image upgrade

### CODE REVIEW

File: os/board/common/Kconfig [hunk 1]
```diff
@@ -31,20 +31,20 @@ config FLASH_MINOR
 
 config FLASH_PART_SIZE
 	string "Flash partition size list (in KBytes)"
-	default "16,48,192,32,512,2400,1536,1536,1400,8,512,"
```
Comment (btheosam) [inline os/board/common/Kconfig]: Confirmed by testing on artik053. Modified values are enough for ssrw partition.

**Comment by seinfra** [comment]
Target : [9cc903b70a55bdfc959dfc2bb1c60abb25c44800] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9cc903b70a55bdfc959dfc2bb1c60abb25c44800] - Code Rule Check OK.

**Comment by btheosam** [comment]
Had to close this as this has to be segregated.

**Pull request closed without merging**